pom.xml

```diff
@@ -37,7 +37,19 @@
 
         <spring.version>3.1.4.RELEASE</spring.version>
     </properties>
-
+  
+    <profiles>
+        <profile>
+           <id>doclint-java8-disable</id>
+            <activation>
+                <jdk>[1.8,)</jdk>
```

Comment:
yes, you're absolutely right:
http://maven.apache.org/enforcer/enforcer-rules/versionRanges.html